### PR TITLE
[user-authn][docs] Add information about password creation in user-authn module

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Added information about creating a password hash sum in the section "The user-auth module: usage"

## Why do we need it, and what problem does it solve?

Currently, there is no information on how to generate a password hash on the page, and this hash amount is in the given config.

## Why do we need it in the patch release (if we do)?

To reduce possible confusion in for a reader.

## What is the expected result?

The module configuration example becomes more understandable for the user.

## Checklist
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Minor documentation updates of the user-authn module.
impact_level: low
```

### CODE REVIEW

File: modules/150-user-authn/docs/USAGE.md [hunk 1]
```diff
@@ -240,6 +240,16 @@ data:
 
 ## An example of creating a static user
 
+Придумайте пароль и укажите его хеш-сумму в поле `password`.
+
+Для вычисления хеш-суммы пароля воспользуйтесь командой:
```
Comment (kirkonru) [inline modules/150-user-authn/docs/USAGE.md:245]:
```suggestion
Create a password and enter its hash in the `password` field.

Use the command below to calculate the password hash:
```

File: modules/150-user-authn/docs/USAGE.md [hunk 2]
```diff
@@ -240,6 +240,16 @@ data:
...
+
+```shell
+echo "$password" | htpasswd -inBC 10 "" | tr -d ':\n' | sed 's/$2y/$2a/'
+```
+
+Также можно воспользоваться [онлайн-сервисом](https://bcrypt-generator.com/).
```
Comment (kirkonru) [inline modules/150-user-authn/docs/USAGE.md:251]:
```suggestion
Alternatively, you can use the [online service](https://bcrypt-generator.com/) to calculate the password hash.
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6023](https://github.com/deckhouse/deckhouse/pull/6023) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6024](https://github.com/deckhouse/deckhouse/pull/6024) to the branch [release-1.51](https://github.com/deckhouse/deckhouse/tree/release-1.51) successful!

**Pull request merged**